### PR TITLE
test: assert for config file content

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
Test update

**Did you add tests for your changes?**
Update `init` test suite.

**If relevant, did you update the documentation?**
N/A

**Summary**
Add checks to make sure that `package.json` and `webpack.config.js` are correctly configured

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]
Thanks

**Comment by jamesgeorge007** [comment]
CI was failing as `require` returned the cached result.

File: test/init/init.test.js [hunk 1]
```diff
@@ -276,7 +278,7 @@ describe('init command', () => {
         });
 
         // Check if package.json is correctly configured
-        const pkgJson = require(join(assetsPath, 'package.json'));
+        const pkgJson = readFileContent(join(assetsPath, 'package.json'));
         expect(pkgJson.scripts['build'] === 'webpack --mode=production').toBeTruthy();
         expect(pkgJson.scripts['serve'] === 'webpack serve').toBeTruthy();
```
Comment (alexander-akait) [inline test/init/init.test.js:283]: Do we have command for build in `dev` mode?

Comment (jamesgeorge007) [reply test/init/init.test.js:283]: Nope

Comment (alexander-akait) [reply test/init/init.test.js:283]: Let's do it in separate PR, i think ideally will be great to have:
- `build`
- `build:dev`
- `build:prod` (same as `build`)
- `serve`
- `watch` (just watching without dev server)

**Review by alexander-akait — DISMISSED** [review]

**Review by alexander-akait — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**